nutkit/frontend/bookmark_manager.py

```diff
@@ -20,22 +24,60 @@ class Neo4jBookmarkManagerConfig:
 
 @dataclass
 class BookmarkManager:
```

Comment:
I agree that the concerns should be separated. I wish I also could refactor the `ResolverResolutionRequired` and `DomainNameResolutionRequired` registries into their own class. But that wouldn't work because they don't send the driverId they were registered with :disappointed: